src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Why would the user have to use JS to add a class? 
Here's how I thought users would do this:

**Vanilla**:
```html
<div class="bx--breadcrumb">
  ...
  <div class="bx--breadcrumb-item bx--breadcrumb-item--last">
    <a href="#" class="bx--link">Breadcrumb 3</a>
  </div>
</div>
```

**React**:
```jsx
<Breadcrumb>
...
  <BreadcrumbItem href="www.google.com" className="bx--breadcrumb-item--last">
    Breadcrumb 3
  </BreadcrumbItem>
</Breadcrumb>
```